File: CSC/TestSumPrimesCpp/SumPrimes.cpp
int sumPrimes(int n) {
    int sum = 0;
    for ( int i=1; i < n; i++ ) {  // loop over possible primes
        bool prime = true;
        for (int j=1; j < 10; j++) { // loop over possible factors
            if (i % j == 0) prime = false;
        }
        if (prime) sum += i;
    }
    return sum;
}

File: CSC/TestSumPrimesPy/SumPrimes.py
import math
import sys

def sumPrimes(n) :
    sum = 0;
    for i in range(1, n):   # loop over possible primes
        prime = True;
        for j in range(1, 10):  # loop over possible factors
            if (i % j == 0) : prime = False;
        if (prime) : sum += i;
    return sum;
